managed-ledger/src/main/java/org/apache/bookkeeper/mledger/LedgerOffloader.java

```diff
@@ -85,6 +161,32 @@
                                     UUID uid,
                                     Map<String, String> extraMetadata);
 
+    /**
+     * Offload the passed in ledger to longterm storage.
+     * Metadata passed in is for inspection purposes only and should be stored
+     * alongside the segment data.
+     *
+     * When the returned future completes, the ledger has been persisted to the
+     * loadterm storage, so it is safe to delete the original copy in bookkeeper.
```

Comment:
Please check the comment of this method, I think here is not correct, The result just return a future for the OffloadHandle, this does not mean that the data has been persisted

Reply:
Comment updated